.github/workflows/test.yml

```diff
@@ -519,6 +519,8 @@ jobs:
             test_prefix: test/clt-tests/plugins/
           - name: Buddy
             test_prefix: test/clt-tests/buddy/
+          - name: Update
+            test_prefix: test/clt-tests/optimisation-and-update/
```

Comment:
No `**` here in the end. Is it ok?

Reply:
This file apparently runs `**` . Only folders are specified, and tests in them will be run.